test/test_browser.py

```diff
@@ -3730,6 +3730,7 @@ def test_dynamic_link_many(self):
     self.run_process([EMCC, 'side2.c', '-sSIDE_MODULE', '-o', 'side2.wasm'])
     self.btest_exit(self.in_dir('main.c'), args=['-sMAIN_MODULE=2', 'side1.wasm', 'side2.wasm'])
 
+  @requires_threads
```

Comment:
Unrelated obvious one-liner. I could open a separate PR if you prefer @sbc100

Reply:
Is this needed anymore?  I think maybe not?   Can we just remove `@requires_threads` and the corresponding `EMTEST_LACKS_THREAD_SUPPORT`?

Reply:
I still use it locally (because the threading tests can be slow/resource-intensive). But maybe that's not a strong reason to keep it?